binary/asn1/schema.asn.value

```diff
@@ -0,0 +1,203 @@
+value Configuration ::= {
+  harts {
+      {
+        hartid {{single 1, start 0, end 0}},
+        debug {
+          trigger {
+            {
+              index {start 0, end 3},
+              mcontrol {
+                {
+                  maskmax  4,
+                  hit TRUE,
+                  addressMatch TRUE,
+                  dataMatch TRUE,
+                  timingBefore TRUE,
+                  timingAfter TRUE,
+                  sizeAny TRUE,
+                  sizeS8 TRUE,
+                  sizeS16 TRUE,
+                  sizeS32 TRUE,
+                  sizeS64 TRUE,
+                  sizeS80 TRUE,
+                  sizeS96 TRUE,
+                  sizeS112 TRUE,
+                  sizeS128 TRUE,
+                  actionMmode TRUE,
+                  actionDmode TRUE,
+                  chain TRUE,
+                  matchEqual TRUE,
+                  matchNapot TRUE,
+                  matchGreaterEqual TRUE,
+                  matchLess TRUE,
+                  matchLowMask TRUE,
+                  matchHighMask TRUE,
+                  matchNotEqual TRUE,
+                  matchNotNapot TRUE,
+                  matchNotLowMask TRUE,
+                  matchNotHighMask TRUE,
```

Comment:
In the other example these are false, and some others are as well.

Reply:
I will check it over again.